beans/Trial.java

```diff
@@ -0,0 +1,40 @@
+package beans;
+
+public class Trial {
+    protected static final int REQUIRED_MARK_TO_PASS = 50;
+    private final String account;
+    private int mark1;
+    private int mark2;
+
+    public Trial(String account, int mark1, int mark2) {
+        this.account = account;
+        this.mark1 = mark1;
+        this.mark2 = mark2;
+    }
+
+    @Override
+    public String toString() {
+        return account + ";" + Integer.toString(mark1) + ";" + Integer.toString(mark2);

```

Comment:
```suggestion
        return account + ";" + mark1 + ";" + mark2;
```
add simple class name in the beginning.